packages/ci-v5/lib/utils.js

```diff
@@ -21,22 +21,7 @@ async function getPipeline(context, client) {
 // Deep get in an object, returning undefined if the path is invalid
 // e.g. get([{ foo: { bar: 'baz' } } ], 0, 'foo', 'bar') => 'baz'
 //
-function dig(obj, ...path) {
```

Comment:
This duplicates the functionality of lodash/get

Reply:
This makes me a little nervous, are you sure there is no difference?

Reply:
Interface differences exist. `get` takes different inputs formats for the path and allows for default returns. 
`dig` was used in 1 location, and the command would fail if this weren't working.